### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Hassaanjbaig-code**
## In this pull request 

- Creating a page 
   - [Splash screen](https://github.com/microverseinc/curriculum-rails/blob/cb603d801b490e0ed1080bce2de297beca82da23/capstone/rails_capstone.md?plain=1#L65-L66)
   - [Sign up and log in pages](https://github.com/microverseinc/curriculum-rails/blob/cb603d801b490e0ed1080bce2de297beca82da23/capstone/rails_capstone.md?plain=1#L68-L71)
   - [categories page](https://github.com/microverseinc/curriculum-rails/blob/cb603d801b490e0ed1080bce2de297beca82da23/capstone/rails_capstone.md?plain=1#L73-L77)
   - [Transactions page](https://github.com/microverseinc/curriculum-rails/blob/cb603d801b490e0ed1080bce2de297beca82da23/capstone/rails_capstone.md?plain=1#L79-L83)
   - ["Add a new category" page](https://github.com/microverseinc/curriculum-rails/blob/cb603d801b490e0ed1080bce2de297beca82da23/capstone/rails_capstone.md?plain=1#L85-L88)
   - ["Add a new transaction" page](https://github.com/microverseinc/curriculum-rails/blob/cb603d801b490e0ed1080bce2de297beca82da23/capstone/rails_capstone.md?plain=1#L90-L94)
   - Creating a back button 

- Also Write a unit and integration test ✔ 
Also for integration testing you should update email and password already created user name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,78 +55,61 @@ After you're finished please remove all the comments and instructions!
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](https://budget-app-385w.onrender.com)
```
Comment (VICTOR-CHUKWU) [inline README.md]: - [ ] You did a nice job but your live link is not same as the app running locally, kindly fix this

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Hassaanjbaig-code , 

## STATUS: CHANGES REQUIRED :recycle: 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by AshakaE — APPROVED** [review]
Hi @Hassaanjbaig-code , 

### Highlights ✨ 

- Thanks for making the requested changes 👏 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Hassaanjbaig-code** [comment]
@AshakaE Thank you so much

**Pull request merged**